test/index.js

```diff
@@ -6,12 +6,7 @@ var src = umd('sentinel-prime', 'return "sentinel"')
 var namespacedSrc = umd('sentinel.prime', 'return "sentinel"')
 var multiNamespaces = umd('a.b.c.d.e', 'return "sentinel"')
 var dollared = umd('$', 'return "sentinel"')
-var number = umd('2sentinel', 'return "sentinel"')
-var strip = umd('sentinel^', 'return "sentinel"')
 
-it('throws when no valid identifier is produced', function () {
-  assert.throws(umd.bind(null, '^!', ''), /Invalid JavaScript/)
-})
```

Comment:
@jazzzz Too long ago to remember my thinking ;)

If it passes by all means hang onto it